### DISCUSSION TITLE
Use usbipd with USB device forwarded via Remote Desktop

### DISCUSSION THREAD

**Discussion by tinloaf** [dorssel/usbipd-win#664] [Q&A]
Hi,

I am using `usbip` to forward a Yubikey into my WSL2 container. It works great when I actually sit at the machine running the WSL2 container.

However, I also often use that machine via Remote Desktop (i.e., RDP). By some magic (maybe even also USB/IP?), Remote Desktop manages to forward my Yubikey to the host machine, i.e., I do see the Yubikey in Windows, I can run all of Yubikey's own tools, I can use it as a OpenPGP smartcard in Kleopatra, etc.

However, it does not appear when running `usbipd list`. I guess that `usbipd` only sees devices physically connected to the system? Is there any way to also forward devices which are already forwarded through RDP? 

Thanks!

**Comment by dorssel** [comment]
The way Remote Desktop forwards the device does not make it a USB device on the remote. As `usbipd-win` can only handle USB devices, it cannot use this "proprietry forwarded device". If the device would be like a "simulated USB device", then there is nothing preventing `usbipd-win` to forward (like: chain-forwarding) the device even further.
